expo-maps/android/src/main/java/expo/modules/maps/ExpoGoogleMapsModule.kt

```diff
@@ -18,6 +18,22 @@ class ExpoGoogleMapsModule : Module() {
       prop("mapType") { view: GoogleMapsView, mapType: MapType ->
         view.setMapType(mapType)
       }
+      prop("showZoomControls") { view: GoogleMapsView, enabled: Boolean ->
+        view.setShowZoomControlButton(enabled)
```

Comment:
```suggestion
        view.setShowZoomControls(enabled)
```

Maybe we should try to rename functions to match prop names. What do you think? Overall everything looks good here, it's just cosmetic change suggestion.